### ISSUE TITLE
Falsy values in frontmatter cannot be used in custom components

### ISSUE THREAD

**Issue by benoitdemaegdt** [nuxt/content#721]
Hello 👋 

First of all, thank you all for this amazing module 🚀 

I think I noticed a bug happening when a variable in the frontmatter is falsy : it cannot be passed to custom components.

When a markdown file looks like this 👇 
```
---
comments: 0
---
# My first blog post

Welcome to my first blog post using content module

<custom-component :comments="comments"><custom-component/>
```

You get this error 👇 

![image](https://user-images.githubusercontent.com/25749578/104886396-f44c2080-5969-11eb-8cd9-a7ea91a4aeda.png)

### Version
@nuxt/content: 1.11.1
nuxt: 2.14.12

### Reproduction Link
https://github.com/benoitdemaegdt/nuxtdemo

### Steps to reproduce
- git clone the github repository above
- `npm i`
- `npm run dev`
- visit : `http://localhost:3000/blog/my-first-blog-post`

### What is Expected?
I expect the falsy value to be passed to the custom component (same behavior as truthy values).
In the example above, the props **comments** in CustomComponent.vue would be equal to 0 and the app does not crash.

### What is actually happening?
There is either a `<VAR_NAME> is not defined` error or a **ReferenceError**, the app crashes 👇 
![image](https://user-images.githubusercontent.com/25749578/104886611-54db5d80-596a-11eb-8f28-66ac454db680.png)

**Comment by AlexanderBelokon** [comment]
Hi! This was recently fixed in https://github.com/nuxt/content/pull/685

**Comment by benoitdemaegdt** [comment]
So cool !  🎉 
This fix is not included in @nuxt/content latest version (v 1.11.1). Any idea about the date of the next release ?

**Comment by AlexanderBelokon** [comment]
Sorry, no(
